src/button.js

```diff
@@ -1,7 +1,15 @@
-export function createButton(node, text, callback){    
-    let textContainer = document.createElement('div');
-    textContainer.innerText = text;    
-    node.classList.add('my-button');
-    node.appendChild(textContainer);
-    node.addEventListener('click', callback);
+export class MyButton {
+    create(target, creationOptions) {
+        target.classList.add('my-button');
+
+        if(!!creationOptions.text) {

```

Comment:
No need to use `!!`. Condition in `if` expression will convert to boolean automatically.